src/partials/footer-scripts.hbs

```diff
@@ -17,6 +17,28 @@
 <script defer src="{{#unless preview}}{{{uiRootPath}}}/js/vendor/{{/unless}}fontawesome-icon-defs.js"></script>
 <script defer src="{{{uiRootPath}}}/js/vendor/fontawesome.js"></script>
 <script async src="{{{uiRootPath}}}/js/vendor/highlight.js"></script>
+
+{{#with (resolvePage page.relativeSrcPath model=false)}}
+    {{#unless (eq ./asciidoc.attributes.stem undefined)}}
```

Comment:
The convention we've been following is to not indent logic tags. It creates superfluous nesting in the resulting HTML.